install/update.rst

```diff
@@ -93,6 +93,11 @@ Updating Zammad
 
    .. tab:: Source
 
+      .. danger::
+
+         Zammads scheduler script has changed! Please ensure to reinstall
```

Comment:
Zammad's former `scheduler.rb` script has changed and is now called `background-worker.rb`.